qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -63,20 +64,27 @@ def circuits(self):
 
         sub_circuits = []
         sub_qubits = []
+        sub_maps = []
         sub_size = []
         num_qubits = 0
 
         # Generate data for combination
-        for expr in self._experiments:
+        for sub_exp in self._experiments:
+
+            # Generate transpiled subcircuits
+            circuits = sub_exp._transpiled_circuits()
+
             # Add subcircuits
-            circs = expr.circuits()
-            sub_circuits.append(circs)
-            sub_size.append(len(circs))
+            sub_circuits.append(circuits)
+            sub_size.append(len(circuits))
 
-            # Add sub qubits
-            qubits = list(range(num_qubits, num_qubits + expr.num_qubits))
+            # Sub experiment logical qubits in the combined circuits full qubits
+            qubits = list(range(num_qubits, num_qubits + sub_exp.num_qubits))
```

Comment:
This logic is based on the implicit assumption that every experiment has different qubits. This should be explicitly validated in the constructor, at line 60.

Reply:
This already is validated in BaseExperiment which raises an exception if there are duplicate qubits in the qubit arg, which there will be if any component experiments are defined on overlapping qubits

Reply:
Thanks you're right. I just missed the logic there.